packages/flutter_tools/lib/src/device.dart

```diff
@@ -948,6 +952,7 @@ class DebuggingOptions {
       if (route != null) '--route=$route',
       if (platformArgs['trace-startup'] as bool? ?? false) '--trace-startup',
       if (enableImpeller) '--enable-impeller',
+      if (serveObservatory) '--serve-observatory',
```

Comment:
I'm guessing today this is a no-op since the Dart SDK defaults to this on anyway?

Reply:
That's correct.